R/importWQP.R

```diff
@@ -57,8 +57,9 @@ importWQP <- function(obs_url, zip=TRUE, tz=""){
   if(!file.exists(obs_url)){
     
     if(zip){
+      message("zip encoding access still in development")
       temp <- tempfile()
-      # options(timeout = 120)
+      options(timeout = 120)
```

Comment:
I have no idea what the difference is here, but I think timeout refers to the readTimeout (how long after no data has been received until stopping).  Both probably do the same thing, but options may apply to other packages.